source/sync/app-builder/local-to-sync.txt

```diff
@@ -125,3 +110,13 @@ App Logic
 
 #. On each subsequent app load, check if the local realm has been
    deleted.
+
+.. figure:: /images/local-to-sync-data-copy.png
```

Comment:
Very minor thing in the diagram. In the first edge case, there are two periods where I'd expect one or three.
<img width="249" alt="Screenshot 2023-08-15 at 10 12 56 AM" src="https://github.com/mongodb/docs-app-services/assets/115574589/1bb0b1dd-fbe1-45f9-8bd0-1fdea03adc70">